wtxmgr/tx_test.go

```diff
@@ -54,17 +55,43 @@ func testDB(t *testing.T) (walletdb.DB, error) {
 	tmpDir := t.TempDir()
 	db, err := walletdb.Create(
 		"bdb", filepath.Join(tmpDir, "db"), true, defaultDBTimeout,
+		false,
 	)
 	return db, err
 }
 
+// exampleDB creates a new database in a temporary directory.
+//
+// NOTE: THis is used for golang example functions.
+func exampleDB() (walletdb.DB, func(), error) {
+	tmpDir, err := os.MkdirTemp("", "example_*")
+	if err != nil {
+		return nil, nil, err
+	}
+
+	db, err := walletdb.Create(
+		"bdb", filepath.Join(tmpDir, "db"), true, defaultDBTimeout,
+		false,
+	)
+	if err != nil {
+		os.RemoveAll(tmpDir)
+		return nil, nil, err
+	}
+
+	return db, func() {
+		db.Close()
```

Comment:
nit: both `Close()` and `RemoveAll(..)` returns an error we could check (`require.NoError`). Same elsewhere.